core/evtforward/ethereum/config.go

```diff
@@ -30,12 +30,14 @@ type Config struct {
 	// Level specifies the logging level of the Ethereum implementation of the
 	// Event Forwarder.
 	Level                  encoding.LogLevel `long:"log-level"`
+	MaxEthereumBlocks      uint64            `long:"max-ethereum-blocks"`
 	PollEventRetryDuration encoding.Duration
 }
 
 func NewDefaultConfig() Config {
 	return Config{
 		Level:                  encoding.LogLevel{Level: logging.InfoLevel},
 		PollEventRetryDuration: encoding.Duration{Duration: defaultDurationBetweenTwoRetry},
+		MaxEthereumBlocks:      10000,
```

Comment:
_very_ pedantic: much like `defaultDurationBetweenTwoRetry` (which probably should be `Retries`), maybe a const instead of a hard-coded value to just move the default value as close to the top of the file as possible, where we can add comments on why it's a certain value (which I know is arbitrary ATM, but kind of based on a common default).